### PR TITLE
new_tests_30_09

### PR DESCRIPTION
**Pull request by muroslav2909**
Please, do not merge it. 
Just view and tell me you opinion. Which way looks better, I I will follow it.

### CODE REVIEW

**Review by epogrebnyak — APPROVED** [review]
Please brign the v2 of test fucntion to issue, will discuss that later on there.

**Comment by epogrebnyak** [comment]
Should I comment now or there some work going on? - #28

**Comment by muroslav2909** [comment]
@epogrebnyak the tests for brent are finished. I am waiting for answer https://github.com/mini-kep/parsers/issues/15 how to go, and will make some edits in test_brew.py after it.

Now I'm working on test_cbr_fx, and will push here a little bit later.

File: parsers/getter/brent.py [hunk 1]
```diff
@@ -10,7 +10,10 @@
 def format_string(date_string):
     return datetime.strptime(date_string, "%Y%m%d").strftime("%Y-%m-%d")
 
-
+# MS: can't understand why we have assert inside a codebase ?
+# EP: please mark your comments initials, so that it is tracable who is writing 
+#     the reason for this assert is that it should be converted to test
+#     better way os saying that is TODO(MS): convert to test
```
Comment (muroslav2909) [inline parsers/getter/brent.py]: @epogrebnyak got it. I did the test for this here https://github.com/mini-kep/parsers/pull/27/files#diff-420a956e849c55a144dfe7e6f34d13acR54

I will remove this comments in code, And assert too.

File: parsers/getter/brent.py [hunk 2]
```diff
@@ -24,7 +27,8 @@ def make_url(access_key=EIA_ACCESS_KEY):
 def fetch(url):
     r = requests.get(url)
     return r.text
-    return json.loads(r.text)
+    # EP: why json.loads(r.text) will not execute?
```
Comment (muroslav2909) [inline parsers/getter/brent.py:24]: @epogrebnyak Method will make exit after first return. Will be executed only first one

**Comment by muroslav2909** [comment]
@epogrebnyak pls, make review

**Pull request merged**